.github/workflows/Windows-CI.yml

```diff
@@ -22,80 +19,144 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - os:             windows-2019
-            cmakeGenerator: VS16Win64
-          - os:             windows-2022
-            cmakeGenerator: Ninja
+          - os:              windows-2019
```

Comment:
Are we really testing all these variants? It may be a bit excessive IMO.

Reply:
Good point. We can probably cut back a bit.

Reply:
let's just stick to 1 Windows build for the time being; that should at least get the ball rolling.

Reply:
How about 2 Windows builds? One on Windows (Server) 2019, and one on Windows (Server) 2022. I think that will be enough even to satisfy me, haha.

Reply:
@stephengtuggy if we need to

Reply:
well, VS2022 on Windows 2022 passed.